src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveStreams.Core.verified.txt

```diff
@@ -1619,6 +1619,7 @@ namespace Akka.Streams.Dsl
         Akka.Streams.Dsl.IRunnableGraph<TMat2> MapMaterializedValue<TMat2>(System.Func<TMat, TMat2> func);
         Akka.Streams.Dsl.IRunnableGraph<TMat> Named(string name);
         TMat Run(Akka.Streams.IMaterializer materializer);
+        TMat Run(Akka.Actor.ActorSystem actorSystem);
```

Comment:
All of the `IRunnableGraph` methods have been updated to accept an `ActorSystem` in addition to an `IMaterializer`